### PR TITLE
Created collections report sample program (C#)

### PR DESCRIPTION
**Pull request by nicoleajoy**
result

### CODE REVIEW

File: CsharpExample/CollectionsReport/Results.csv [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+d6b8c0e5-7b2c-41d5-98d5-f6f1663c0ac8,07/10/2021 00:00:00,3587,Carlos
+f7dcf7e1-979e-4bea-bb69-277c67c00e3a,07/10/2021 00:00:00,3860,Gibby
+7d75fb17-d3f5-44df-a9bd-6e414e43bb9c,07/10/2021 00:00:00,3292,Catherine
+5c5953b9-bce1-4e2f-aa47-a85005db0020,07/10/2021 00:00:00,2100,Marisol
+010d3990-1b24-4bd0-8c5c-7a3d2ac8d43d,07/10/2021 00:00:00,1807,Max
+7ecb7613-5f41-4fa7-aa8c-a95ad8c5ee46,07/10/2021 00:00:00,2246,Catherine
+d74565f3-957e-4a3b-bd91-510d23dc505a,07/10/2021 00:00:00,3900,Ted
+a3487ab9-6c5e-4121-ae83-1f8c69906290,07/10/2021 00:00:00,1474,Stephen
+73a9e127-4a14-4836-afb1-a300860f0959,07/10/2021 00:00:00,645,Shane
+e949b9ea-6e56-43d9-a5b1-63868b20eb15,07/10/2021 00:00:00,1214,Shane
```
Comment (tspence) [inline CsharpExample/CollectionsReport/Results.csv]: Let's leave this file out of the PR.  It's just demo data.

**Comment by tspence** [comment]
LGTM!  Any other changes?

File: CsharpExample/CollectionsReport/Results.csv (hunk 1 above)
Comment (nicoleajoy) [reply CsharpExample/CollectionsReport/Results.csv]: Sounds good!

**Comment by nicoleajoy** [comment]
Great! That should be all, thank you.

**Comment by tspence** [comment]
LGTM!

**Pull request merged**